Game/character.py

```diff
@@ -0,0 +1,77 @@
+class Archer(object):
+
+	def __init__(self, name, weapon, armor, hitpoints = 75):
+		self.name = name
+		self.weapon = weapon
+		self.armor = armor
+		self.type = "archer"
+		self.hitpoints = hitpoints
+
+	def pickUpWeapon(self, weapon):
+		if weapon.damage >= self.weapon.damage:
+			if weapon.weapon in ("bow", "long bow", "cross bow"):
+				print "\nYou pick up the enemy's %s" % weapon.weapon
+				self.weapon = weapon
+			else:
+				print "You can not use the %s, since you don't posses the skills to use it." % weapon.weapon
+		else:
+			print "You decide to leave the %s there as it is inferior to your %s" % (weapon.weapon, self.weapon.weapon)
+
+	def pickUpArmor(self, armor):
+		if self.armor <= armor.protection:
+			print "\nYou pick up the enemy's %s" % self.armor.armor
+			self.armor = armor
+		else:
+			print "Your armor is superior to the enemy's armor, so you leave it there."
+
+class Swordsman(object):
+
+	def __init__(self, name, weapon, armor, hitpoints = 100):
+		self.name = name
+		self.weapon = weapon
+		self.armor = armor
+		self.type = "swordsman"
+		self.hitpoints = hitpoints
+
+	def pickUpWeapon(self, weapon):
+		if weapon.damage >= self.weapon.damage:
+			if weapon.weapon in ("sword", "short sword", "long sword"):
+				print "\nYou pick up the enemy's %s" % weapon.weapon
+				self.weapon = weapon
+			else:
+				print "You can not use the %s, since you don't posses the skills to use it." % weapon.weapon
+		else:
+			print "You decide to leave the %s there as it is inferior to your %s" % (weapon.weapon, self.weapon.weapon)
+
+	def pickUpArmor(self, armor):
```

Comment:
I notice you have the same pickUpArmor and pickUpWeapon in many of the classes.  It's best to create a class where you inherit them.

Also, since pickUpWeapon depends on the certain class's weapon such as Swordsman using sword and Thief using dagger.  It's better to create an array of weapons such as self.weapons = ['sword', 'short sword', 'long sword'] and then do an if on that. That way you can have generic functions and the attributes will be the one with different values.